src/networks.js

```diff
@@ -21,24 +15,39 @@ export const networkConfigs = {
     network_reserve: '0xec0dd1579551964703246becfbf199c27cb84485',
     nodes: {
       defaultEth: 'https://mainnet.eth.aragon.network/',
+      subgraph: {
+        http:
+          'https://graph.backend.aragon.org/subgraphs/name/aragon/aragon-court',
+        ws: 'wss://graph.backend.aragon.org/subgraphs/name/aragon/aragon-court',
```

Comment:
Just wondering, I think we've removed the websocket endpoints because they're not working reliably with subgraphs at the moment (#381). Are we able to remove these as well?

Reply:
are the normal subscriptions (without the polling query) working with your infra?

Reply:
Oh yeah true, i think we can remove them yes.

> are the normal subscriptions (without the polling query) working with your infra?

Maybe they are, but we currently we are using their node for mainnet and for testnets we are using the subgraph's original one.

Reply:
Done in https://github.com/aragon/court-dashboard/pull/391